devtools-html/debugger.html
# Update source list when back and forward buttons are clicked

Associated Issue: #635

### Summary of Changes
* Reload debuggee when a back/forward button is clicked. 

### Test Plan
1. Open debugger in Chrome
2. Open debuggee in Firefox
3. Go to https://devtools-html.github.io/debugger-examples/examples/todomvc/ in the debugee
4. Go to https://www.cnet.com/
5. Observe Sources
6. Click back button 
7. Observe Sources
8. Click forward button
9. Observe Sources
